jsx/Form.js

```diff
@@ -783,14 +850,58 @@ var ButtonElement = React.createClass({
   render: function() {
     return (
       <div className="row form-group">
-        <div className="col-sm-9 col-sm-offset-3">
-          <button type={this.props.type}
-                  className="btn btn-primary"
-                  onClick={this.handleClick}>
+        <div className={this.props.columnSize}>
+          <button
+            type={this.props.type}
+            className={this.props.buttonClass}
+            onClick={this.handleClick}
+          >
             {this.props.label}
           </button>
         </div>
       </div>
     );
   }
 });
+
+/**
+ * Generic form element.
+ */
+var LorisElement = React.createClass({
+
+  render: function() {
+    var props = this.props.element;
```

Comment:
Can we call this `elementProps` instead of just `props`? Or just something to differentiate it from this component's `props`.